numpy/random/mtrand/mtrand.pyx

```diff
@@ -2885,28 +2911,27 @@ cdef class RandomState:
         .. math:: f(x; \\mu, \\lambda) = \\frac{1}{2\\lambda}
                                        \\exp\\left(-\\frac{|x - \\mu|}{\\lambda}\\right).
 
-        The first law of Laplace, from 1774, states that the frequency of an error
-        can be expressed as an exponential function of the absolute magnitude of
-        the error, which leads to the Laplace distribution. For many problems in
-        Economics and Health sciences, this distribution seems to model the data
-        better than the standard Gaussian distribution
-
+        The first law of Laplace, from 1774, states that the frequency
+        of an error can be expressed as an exponential function of the
+        absolute magnitude of the error, which leads to the Laplace
+        distribution. For many problems in economics and health
+        sciences, this distribution seems to model the data better
+        than the standard Gaussian distribution.
 
         References
         ----------
-        .. [1] Abramowitz, M. and Stegun, I. A. (Eds.). Handbook of Mathematical
-               Functions with Formulas, Graphs, and Mathematical Tables, 9th
-               printing.  New York: Dover, 1972.
+        .. [1] Abramowitz, M. and Stegun, I. A. (Eds.). "Handbook of
+               Mathematical Functions with Formulas, Graphs, and Mathematical
+               Tables, 9th printing," New York: Dover, 1972.
 
```

Comment:
Looks like most of the reference sections don't have blank lines between the entries.